cardano-cli/src/Cardano/CLI/EraBased/Options/Common.hs

```diff
@@ -1406,11 +1406,14 @@ pPlutusStakeReferenceScriptWitnessFiles prefix autoBalanceExecUnits =
     <*> pure Nothing
 
 pPlutusScriptLanguage :: String -> Parser AnyScriptLanguage
-pPlutusScriptLanguage prefix =
-  Opt.flag' (AnyScriptLanguage $ PlutusScriptLanguage PlutusScriptV2)
-    (  Opt.long (prefix ++ "plutus-script-v2")
-    <> Opt.help "Specify a plutus script v2 reference script."
-    )
+pPlutusScriptLanguage prefix = plutusP PlutusScriptV2 "v2" <|> plutusP PlutusScriptV3 "v3"
```

Comment:
should we still support v2 here?

Reply:
Yes and we should technically still support `PlutusScriptV1` as well. I'm not sure when or why that was dropped because V1 scripts still have to work on mainnet.